HACKING

```diff
@@ -341,6 +341,22 @@ Example::
     {
         ...
 
+Translation
+^^^^^^^^^^^
+
+Strings which are visible to geany users need to be translated and thus
+should be enclosed in ``_( )`` to have them being recognized by the GNU
+translation tools. Also spaces right after the first ``"`` of a string
+shall not be used as they will be cut of by the tools. This is a bad example::
```

Comment:
No they won't be cut off.  If you're referring to my erroneous comment on @Skif-off's PR recently, it was just me reading the diff the wrong way around and thinking he was removing the space instead of adding it.
GNU GetText properly handle any C string, so where to place the space is not a concern for it to appear, it's a mere code style thing.